frontend/src/main/web/src/components/Base/readme.md

```diff
@@ -0,0 +1,29 @@
+The base component for most other components. It allows other components to
+inherit it's props, which are useful for styling with atomic classes.
+
+It should never be used directly, only through another component.
+
+## Example of Base in <Icon />
+
+`<Icon />` uses <Base /> as it's core component.
+
+### Theme
+
+Theme should match the Icon theme objects structure.
+
+    <Icon name='project' size='n1'
+      theme={{base: {c: 'C(muted)'}}} />
+
+### Atomic
+
+Atomic class objects for quick style changes.
+
+    <Icon name='project' size='2' atomic={{c: 'C(pri)'}}/>
+
+### Atomic & Theme
+
+Atomic classes can override any theme object **and custom props**.
+
+    <Icon name='project' size='4'
+      theme={{base: {c: 'C(pri)'}}}
+      atomic={{c: 'C(muted)', h: 'H(ms8)', w: 'W(ms8)'}} />
```

Comment:
This is not enough information for someone to easily understand how `theme` and `atomic` work and what all the codes in them mean. This document should have a reference to more thorough information about this, or it should be more thorough itself.

The examples need additional explanation of what the final result will be from what is shown. Particularly in the last example, what are the final styles that would be shown?